integration-tests/features/test_execution_statistics.feature

```diff
@@ -305,74 +218,6 @@ Feature: Providing test execution numbers
               #| test_success_density | 0     | -> enable when this is restored in core
               | test_execution_time  | 3     |
               
-              
-  Scenario Outline: Importing unchanged boosttest reports in detailed mode (filename tag)
```

Comment:
detail mode is gone, this was testing that. from what i see there is still lacking on server side to display at least the test names. from the java plugin they use this still: https://github.com/SonarSource/sonar-java/blob/master/java-surefire/src/main/java/org/sonar/plugins/surefire/SurefireJavaParser.java#L157 

however if i look at the https://github.com/SonarSource/sonarqube/blob/master/sonar-plugin-api/src/main/java/org/sonar/api/test/MutableTestPlan.java#L23 its seems to be going away. 

So likely test metrics will become only numbers and no details. 

#911 to track this